src/lhm.net/Migrator.cs

```diff
@@ -29,24 +28,32 @@ public Migrator(Table origin, IDbConnection connection)
             _dateTimeStamp = DateTime.UtcNow.ToString(Constants.DateFormat);
         }
 
-        public string Name
+        public string Destination
```

Comment:
Yeah I just figured that Migration.Destination was more descriptive than Name.  We could make it LhmTableName what do you think.